bookwyrm/templates/403.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'layout.html' %}
+{% load i18n %}
+{% load utilities %}
+
+{% block title %}{% trans "Oh no!" %}{% endblock %}
+
+{% block content %}
+<div class="block">
+    <h1 class="title">{% trans "Permission Denied" %}</h1>
+    {% blocktrans trimmed with level=request.user|get_user_permission %}
+    <p class="content">You do not have permission to view this page. Your user permission level is <code>{{ level }}</code>.</p>
```

Comment:
Perhaps "to view this page or perform this action"?

Reply:
Yes that's better